<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDebugDlg</class>
 <widget class="QDialog" name="CDebugDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAll">
        <attribute name="title">
         <string>全部信息</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="tBrwsView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDebug">
        <attribute name="title">
         <string>调试信息</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="tBrwsViewDebug"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tEditInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pBtnSession">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>音频设置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pBnOpenLog">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>打开日志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBnLogDir">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>日志文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="cBoxStop">
           <property name="text">
            <string>暂停</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBoxCloseDebug">
           <property name="text">
            <string>关闭调试</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pBnClear">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBnClose">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>关  闭</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pBnSend">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>发  送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../inst.qrc"/>
 </resources>
 <connections/>
</ui>
